examples/c/Makefile

```diff
@@ -38,6 +38,10 @@ else
 	MAKEFLAGS += --no-print-directory
 endif
 
+ifeq ($(ARCH),aarch64)
+	ARCH := arm64
```

Comment:
let's do this in one place, similar to how libbpf-tools (https://github.com/iovisor/bcc/blob/master/libbpf-tools/Makefile#L12) do that:

```
ARCH := $(shell uname -m | sed 's/x86_64/x86/' | sed 's/aarch64/arm64/' | sed 's/ppc64le/powerpc/' | sed 's/mips.*/mips/')
```

Reply:
Sure, seems like these two make files are basically for the same purpose.
I'll align to the BCC version and make a squashed git commit.
Thanks for your comment.